app/views/plans/get_started.html.haml

```diff
@@ -106,11 +106,11 @@
           %h4{class: heading_disabled_class} 4. Do you want to add disease specific planning?
           - if @get_started_form.country.present?
             .form-check
-              = check_box_tag "get_started_form[diseases][]", Plan::DISEASE_TYPES.first, false,
+              = check_box_tag "get_started_form[diseases][]", Disease.influenza.id, false,
                 {class: "form-check-inline",
                   id: "diseases_influenza"}
               %label{for: "diseases_influenza"}
-                Optional: Influenza planning
+                = "Optional: #{Disease.influenza.display} planning"
```

Comment:
This appears to break MVC convention, could we instead perform the fetch of `Disease.influenza` in the controller and set an instance variable that the template uses